widgetpanel.go

```diff
@@ -259,16 +263,21 @@ func (w *widgetPanel) CreateRenderer() fyne.WidgetRenderer {
 	account = widget.NewButtonWithIcon(accountLabel, wmtheme.UserIcon, func() {
 		w.showAccountMenu(account)
 	})
-	batteryIcon := widget.NewIcon(wmtheme.BatteryIcon)
-	brightnessIcon := widget.NewIcon(wmtheme.BrightnessIcon)
-	less := widget.NewButtonWithIcon("", theme.ContentRemoveIcon(), func() {
-		w.setBrightness(-5)
-	})
-	more := widget.NewButtonWithIcon("", theme.ContentAddIcon(), func() {
-		w.setBrightness(5)
-	})
-	bright := fyne.NewContainerWithLayout(layout.NewBorderLayout(nil, nil, less, more),
-		less, w.brightness, more)
+	var brightnessIcon, bright, batteryIcon fyne.CanvasObject
+	if _, err := battery(); err == nil {
```

Comment:
This does not need to be separate from the "if" later when the layout is done does it?
All the vars listed above seem superfluous when you consider that they could be added later when the layout is defined...